src/resources/swagger.yaml

```diff
@@ -842,6 +842,26 @@ paths:
         '200':
           description: 'returns new entity exception after deletion'
 
+  /exception/${programId}/missing-entity:
+    post:
+      tags:
+        - Exception
+      summary: Add submitter donor ids for exception
+      parameters:
+        - $ref: '#/components/parameters/PathProgramId'
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/EntityExceptionDonorIds'
+
+      responses:
+        '201':
+          description: 'successful upload'
+        '401':
+          $ref: '#/components/responses/UnauthorizedError'
```

Comment:
Statuses don't match the request.

Please create an entity for the return type.